@navikt/core/react/src/util/virtualfocus/virtualfocus.stories.tsx

```diff
@@ -0,0 +1,381 @@
+import { Meta } from "@storybook/react";
+import React, {
+  Dispatch,
+  ReactNode,
+  SetStateAction,
+  forwardRef,
+  useRef,
+  useState,
+} from "react";
+import {
+  CloudIcon,
+  HangerIcon,
+  HourglassIcon,
+  HouseIcon,
+  PuzzlePieceIcon,
+  RulerIcon,
+} from "@navikt/aksel-icons";
+import { Button } from "../../button";
+import { Search } from "../../form/search";
+import { VStack } from "../../layout/stack";
+import { Popover } from "../../popover";
+import { useVirtualFocusDescendant } from "./Context";
+import { VirtualFocus } from "./VirtualFocus";
+
+export default {
+  title: "Utilities/VirtualFocus",
+  component: VirtualFocus,
+  decorators: [
+    (Story) => (
+      <VStack gap="10">
+        <Button>above</Button>
+        <Story />
+        <Button>below</Button>
+      </VStack>
+    ),
+  ],
+} as Meta;
+
+const MyAnchor = forwardRef<
+  HTMLInputElement,
+  {
+    children: React.ReactNode;
+    value: string;
+    setValue: Dispatch<SetStateAction<string>>;
+  }
+>(({ children, value, setValue }, ref) => {
+  const { descendants } = useVirtualFocusDescendant();
+  const searchRef = useRef<HTMLInputElement>(null);
+
+  const to_focus = descendants.item(0);
+
+  return (
+    <div style={{ position: "relative" }}>
+      <VirtualFocus.Anchor
+        role="combobox"
+        ref={ref}
+        onSelect={() => {
+          console.log(`onSelect(): ${value}`);
+        }}
+        onActive={() => {
+          console.log(`onActive(): Anchor`);
+        }}
+      >
+        <Search
+          label="Søk på nav sine sider"
+          autoComplete="off"
+          value={value}
+          onChange={(current_input) => {
+            setValue(current_input);
+            to_focus?.handleOnActive();
```

Comment:
What is this for? ` to_focus?.handleOnActive();`

Reply:
I don't remember currently. Might be old unused code that wasn't removed, or copy pasted & forgotten to modify. Either way, it seems to be non-impactful / not useful.

Tested without it, and it changes no behaviour. removed 🔥